doc/direction.md

```diff
@@ -0,0 +1,63 @@
+# Minetest Direction Document
+
+## 1. Background
+
+To give context to this document, you should consider reading the following
+resources:
+
+* celeron55 has published his opinions on the development of Minetest in these places:
```

Comment:
These are not really 'his opinions', the first 2 are actually MT's official roadmaps, just as valid as the priorities we have discussed.
Describing these as 'his opinions' seems to be undermine his authority and the official roadmaps.
Maybe you just worded this badly.

Reply:
Authority is proportional to time invested

Reply:
Celeron55 has far more commits than anyone else, and possibly has done more work than anyone else too. But of course less than the total of everyone else.
But that is partially irrelevant because he is the original creator and occupies a special position.
I hope you do not mean it this way but this implies you are personally deciding to undermine him and personally deciding on the transfer of power. Has this been discussed with and agreed by other core devs? I have not seen any discussion.

Reply:
celeron55 delegated control of development to core developers when he appointed us and stopped actively contributing. His documents are useful in outlining high-level goals, but are outdated and aren't focused. A roadmap is a collection of goals and a plan to obtain these, this certainly matches that

Reply:
That makes some sense, but my concerns still stand about the wording.
We do not have absolute control of development, and this has not been discussed with him and core devs and decided.
His documents are more relevant than you imply, they are old but not outdated, they are the fundamental philosophy of MT.